mod-pubsub-server/src/main/java/org/folio/rest/util/OkapiConnectionParams.java

```diff
@@ -26,7 +26,7 @@ public OkapiConnectionParams(Vertx vertx) {
   public OkapiConnectionParams(Map<String, String> okapiHeaders, Vertx vertx) {
     this.okapiUrl = okapiHeaders.getOrDefault(OKAPI_URL_HEADER, "localhost");
     this.tenantId = okapiHeaders.getOrDefault(OKAPI_TENANT_HEADER, "");
-    this.token = okapiHeaders.getOrDefault(OKAPI_TOKEN_HEADER, "dummy");
+    this.token = okapiHeaders.get(OKAPI_TOKEN_HEADER);
```

Comment:
Will token now be null in those situations when it used to de "dummy"? Can this possibly lead to any side effects, like NPEs in some scenarios?

Reply:
Using null for putHeader and later here is OK.

https://vertx.io/docs/apidocs/io/vertx/core/MultiMap.html#add-java.lang.String-java.lang.String-

The effect is that NO token is sent. Which is exactly what we want.

Reply:
Thanks!